wproofreader/values.yaml

```diff
@@ -31,6 +31,10 @@ certMountPath: "/certificate"
 
 # `dictionaries` structure provide storage for custom user dictionaries inside the cluster
 dictionaries:
+  # enable or disable persistentance of Custom Dictionaries feature
```

Comment:
```suggestion
  # Enable or disable persistence of custom dictionary feature
```

Reply:
enable, not Enable. 
To keep commenting uniform.